Test.cpp

```diff
@@ -103,6 +103,7 @@ int main(int argc, char const *argv[])
   // ---------- End compilers initialization ----------
   // start configuring version v
   builder._compiler = cc;
+  builder._genIROptionList={vc::Option("fpic", "-fPIC")};
```

Comment:
is this really required?

Reply:
I think so, otherwise i get:
```
Notify: v3 IR prepared. Going for v3 compilation
Error: compilation 3 failed.
```
with error
```
/usr/bin/ld: /tmp/opt_IR_.35...a.o: warning: relocation against `global_var' in read-only section `.text'
/usr/bin/ld: /tmp/opt_IR_35...a.o: relocation R_X86_64_PC32 against symbol `global_var' can not be used when making a shared object; recompile with -fPIC
```